frontend/catalyst/jit.py

```diff
@@ -317,6 +329,15 @@ def _get_workspace(self):
 
         return WorkspaceManager.get_or_create_workspace(workspace_name, preferred_workspace_dir)
 
+    def _patch_module_allowlist(self):
+        """Add included modules to module_allowlist."""
+        include_convertlist = ()
+        if self.compile_options.autograph_include is not None:
+            for rule in self.compile_options.autograph_include:
+                include_convertlist = (ag_config.Convert(rule), *include_convertlist)
```

Comment:
Small code improvements:

This for loop can probably be sped up / simplified with a generator comprehension
```suggestion
        include_convertlist = tuple(ag_config.Convert(rule) for rule in self.compile_options.autograph_include)           
```

Additionally, a default value of `()` instead of `None` for `autograph_include` means we can skip the is not None step.

Reply:
Nice trick! Thanks! Done.